### PR TITLE
Describe encrypting secured credentials and presentations

### LINKED ISSUE #175: Add comments on encryption re:OpenID
>  The working group is discussing how we might comment on recent work, that does make use of encryption in https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#section-3.2

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #175

***
<a href="https://pr-preview.s3.amazonaws.com/selfissued/vc-jose-cose/pull/293.html" title="Last updated on Aug 26, 2024, 10:57 PM UTC (f00e0d0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/293/0d29764...selfissued:f00e0d0.html" title="Last updated on Aug 26, 2024, 10:57 PM UTC (f00e0d0)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -263,6 +263,14 @@ <h2 id="securing-with-jose">Securing JSON-LD Verifiable Credentials with JOSE</h
                 A [=conforming JWS verifier implementation=] MUST use [[RFC7515]] to verify [=conforming JWS documents=]
                 that use this media type.
             </p>
+	    <p>
+	      To encrypt the secured [=verifiable credential=]
+	      when transmitted over an insecure channel,
```
Comment (TallTed) [inline index.html:268]:
```suggestion
	      To encrypt a secured [=verifiable credential=]
	      when transmitting over an insecure channel,
```

File: index.html [hunk 2]
```diff
@@ -333,6 +341,14 @@ <h2 id="securing-vps-with-jose">Securing JSON-LD Verifiable Presentations with J
                 Credentials in verifiable presentations MUST be secured.
                 These credentials are secured using JWS in this case.
             <p>
+	    <p>
+	      To encrypt the secured [=verifiable presentation=]
+	      when transmitted over an insecure channel,
```
Comment (TallTed) [inline index.html:346]:
```suggestion
	      To encrypt a secured [=verifiable presentation=]
	      when transmitting over an insecure channel,
```

File: index.html [hunk 3]
```diff
@@ -476,6 +492,14 @@ <h2 id="securing-with-sd-jwt">Securing JSON-LD Verifiable Credentials with SD-JW
                 limited to <a data-cite="VC-DATA-MODEL-2.0#status"><code>credentialStatus</code></a>
                 and <a data-cite="VC-DATA-MODEL-2.0#data-schemas"><code>credentialSchema</code></a>.
             </p>
+	    <p>
+	      To encrypt the secured [=verifiable credential=]
+	      when transmitted over an insecure channel,
```
Comment (TallTed) [inline index.html:497]:
```suggestion
	      To encrypt a secured [=verifiable credential=]
	      when transmitting over an insecure channel,
```

File: index.html [hunk 4]
```diff
@@ -555,6 +579,15 @@ <h2 id="securing-vps-sd-jwt">Securing JSON-LD Verifiable Presentations with SD-J
                 limited to <a data-cite="VC-DATA-MODEL-2.0#status"> <code>credentialStatus</code></a>
                 and <a data-cite="VC-DATA-MODEL-2.0#data-schemas"> <code>credentialSchema</code></a>.
             </p>
+	    <p>
+	      To encrypt the secured [=verifiable presentation=]
+	      when transmitted over an insecure channel,
```
Comment (TallTed) [inline index.html:584]:
```suggestion
	      To encrypt a secured [=verifiable presentation=]
	      when transmitting over an insecure channel,
```

File: index.html [hunk 5]
```diff
@@ -630,6 +663,15 @@ <h2 id="securing-vcs-with-cose">Securing JSON-LD Verifiable Credentials with COS
                 A [=conforming COSE verifier implementation=] MUST use COSE_Sign1 as specified in [[RFC9052]] to verify
                 [=conforming COSE documents=] that use this media type.
             </p>
+	    <p>
+	      To encrypt the secured [=verifiable credential=]
+	      when transmitted over an insecure channel,
```
Comment (TallTed) [inline index.html:668]:
```suggestion
	      To encrypt a secured [=verifiable credential=]
	      when transmitting over an insecure channel,
```

File: index.html [hunk 6]
```diff
@@ -699,6 +741,15 @@ <h2 id="securing-vps-with-cose">Securing JSON-LD Verifiable Presentations with C
                 Credentials in verifiable presentations MUST be secured.
                 These credentials are secured using COSE in this case.
             <p>
+	    <p>
+	      To encrypt the secured [=verifiable presentation=]
+	      when transmitted over an insecure channel,
```
Comment (TallTed) [inline index.html:746]:
```suggestion
	      To encrypt a secured [=verifiable presentation=]
	      when transmitting over an insecure channel,
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor grammar

**Comment by decentralgabe** [comment]
Open for >1 week. Editor's approval. Change requests incorporated. Merging.

**Pull request merged**